app/views/admin/invoices/show.html.erb

```diff
@@ -1 +1,7 @@
-<h3> Admin Invoices Show Page (placeholder)</h3>
\ No newline at end of file
+<section id="invoice-details">
+  <h2><%= "Invoice ##{@invoice.id}"%></h2>
+  <p><%= "Status: #{@invoice.status}"%></p>
+  <p><%= "Created on: #{@invoice.created_at.strftime("%A, %B %d, %Y")}"%></p>
```

Comment:
i was planning on making a method in my model that would handle this date time formatting we may be able to use it here as well